tests/unit/test_indep_task_queues.py

```diff
@@ -73,28 +71,24 @@
     ]
 )
 def test_queue_and_release(
-        active,
-        expected_released,
-        expected_foo_groups):
+    active,
+    expected_released,
+    expected_foo_groups
+):
     """Test task queue and release."""
     # configure the queue
     queue_mgr = IndepQueueManager(QCONFIG, ALL_TASK_NAMES, DESCENDANTS)
 
     # add newly ready tasks to the queue
     for name in READY_TASK_NAMES:
-        itask = Mock()
+        itask = Mock(spec=TaskProxy)
         itask.tdef.name = name
         itask.state.is_held = False
         queue_mgr.push_task(itask)
 
     # release tasks, given current active task counter
     released = queue_mgr.release_tasks(active)
-    assert sorted([r.tdef.name for r in released]) == sorted(expected_released)
-
-    # check released tasks change state to "preparing", and not is_queued
-    for r in released:
-        assert r.state.reset.called_with(TASK_STATUS_PREPARING)
-        assert r.state.reset.called_with(is_queued=False)
```

Comment:
`itask.state.reset()` is only ever called by `TaskPool.release_queued_tasks()`, so not applicable for this test